.github/scripts/create-jacoco-coverage-report.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+set -x
+
+echo "GITHUB_BASE_REF: ${GITHUB_BASE_REF}"
+
+echo "Setting up git remote"
+git remote set-branches --add origin ${GITHUB_BASE_REF}
+git fetch
```

Comment:
let's try to configure the checkout action properly

Reply:
As discussed offline, this doesn't seem possible with the checkout action. Hence, closing this comment.